### ISSUE TITLE
Add API overview page for File API

### ISSUE THREAD

**Issue by wbamberg** [mdn/sprints#2866]
GroupData.json lists "File API" as a Web API with the following contents:

```json
        "File API": {
            "guides":     [ "/docs/Using_files_from_web_applications",
                            "/docs/Extensions/Using_the_DOM_File_API_in_chrome_code" ],
            "interfaces": [ "File",
                            "FileList",
                            "FileReader",
                            "FileReaderSync",
                            "Blob" ],
            "methods":    [],
            "properties": [],
            "events":     []
        },
```

...but it doesn't have an API overview page. We should write a page and add the "overview" property to GroupData.

**Comment by wbamberg** [comment]
@chrisdavidmills , why are these bugs being closed? As far as I can see this page has not been written.

**Comment by chrisdavidmills** [comment]
@wbamberg I'm trying to tidy up the repos a bit, and eventually move everything over to the content repo and close the sprints repo. I thought I'd just close these, but maybe I was a bit overzelous; I'm happy to reopen them and move them to the content repo in you think they'd be useful.

**Comment by wbamberg** [comment]
This overview page isn't written, and GroupData isn't updated. So the reported problem isn't fixed: if we thought it was a problem before it still is IMO. And the same applies to the other issues.

Can we just move these issues to mdn/content?

**Comment by chrisdavidmills** [comment]
I agree with you, but at the same time we have a huge backlog, and some of this stuff has been sitting here not being fixed for years in some cases. I was just trying to make the backlog a bit more manageable. I'll move them to content...
